### PR TITLE
Test Automation For RET-1235 and Cross browser fix

### PR DESCRIPTION
**Pull request by SunnyAyeni**
### JIRA link
https://tools.hmcts.net/jira/browse/RET-1235
https://tools.hmcts.net/jira/browse/RET-1242
### Change description
Test coverage for RET 1242 and 1235
This PR include the automation for RET-1235 and also browser testing for SYA-Frontend

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

File: src/test/functional/supportedBrowsers.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+const supportedBrowsers = {
+  // edge: {
```
Comment (remalauday) [inline src/test/functional/supportedBrowsers.js:2]: are we not doing edge testing Sunday?

File: Jenkinsfile_nightly [hunk 2]
```diff
@@ -15,5 +15,25 @@ def product = "et"
 def component = "sya"
 
 withNightlyPipeline(type, product, component) {
+    env.TEST_URL = params.URL_TO_TEST
+    env.TEST_E2E_URL = params.URL_TO_TEST
```
Comment (remalauday) [inline Jenkinsfile_nightly:19]: can we remove TEST_E2E_URL as we have not used it

File: package.json [hunk 3]
```diff
@@ -20,6 +20,7 @@
     "test:pact:run-and-publish": "yarn test:pact && yarn publish-pact",
     "sonar-scan": "sonar-scanner",
     "test:functional": "NODE_TLS_REJECT_UNAUTHORIZED=0 codeceptjs --config ./src/test/functional/codecept.conf.js run-workers 2 parallel",
```
Comment (marcjudge) [inline package.json:22]: I've been seeing the following message in the logs when install packages "warning " > codeceptjs@3.3.0" has unmet peer dependency "faker@^5.5.1".

Could you please add faker to the package list unless there is a reason you can't?

File: src/test/functional/supportedBrowsers.js (hunk 1 above)
Comment (SunnyAyeni) [reply src/test/functional/supportedBrowsers.js:2]: its not working at this point in time. the plan is to get it working at some point in future

File: package.json (hunk 3 above)
Comment (SunnyAyeni) [reply package.json:22]: will do in the next patchset

File: Jenkinsfile_nightly (hunk 2 above)
Comment (SunnyAyeni) [reply Jenkinsfile_nightly:19]: will do in the next patchset

**Review by marcjudge — APPROVED** [review]

**Review by remalauday — APPROVED** [review]

**Pull request merged**